pyvista/examples/downloads.py

```diff
@@ -8185,3 +8185,38 @@ def _full_head_files_func():  # pragma: no cover
 
 
 _dataset_full_head = _MultiFileDownloadableDatasetLoader(_full_head_files_func)
+
+
+def download_nek5000(load=True):
+    """Download 2D nek5000 data example.
+
+    .. versionadded:: 0.45.0
+
+    Parameters
+    ----------
+    load : bool, optional
+        Load the dataset after downloading it when ``True``.  Set this
+        to ``False`` and only the filename will be returned.
+
+    Returns
+    -------
+    pyvista.Nek5000Reader | str
```

Comment:
This should return a dataset type loaded by the reader, not the reader itself e.g. `pyvista.PolyData`